src/olympia/lib/git.py

```diff
@@ -18,6 +19,8 @@
 
 log = olympia.core.logger.getLogger('z.git_storage')
 
+# A mixture of Blob and TreeEntry
+TreeEntryWrapper = namedtuple('Entry', 'tree_entry, path, blob')
```

Comment:
btw, I'm not happy with that name and open to suggestions. This thing wraps [TreeEntry](https://www.pygit2.org/objects.html#tree-entries) and extends it with some metadata. I wonder if I should just name it `TreeEntry` or just `Entry`?

Reply:
I'm fine with it like this tbh